examples/smart_contracts/v2/javascript/sandbox.js

```diff
@@ -0,0 +1,91 @@
+"use strict";
```

Comment:
can you just post the ts instead? or rewrite the ts in js? 

The generated js is 😢

Reply:
yeah I know this sucks. The ts didn't work so I had to convert ts to js :(

Reply:
copy the file in and rename it with .js extension and see what breaks :)